supabase/migrations/002_profile_insert_policy.sql

```
@@ -0,0 +1,7 @@
+-- ============================================================
+-- 002: profiles INSERT ポリシーの追加
+-- ============================================================
+-- getOrCreateProfile フォールバック（DB トリガー未実行時）が
+-- supabase クライアント経由で自分の profile を作成できるようにする
+CREATE POLICY "profiles: 自分のプロフィールを作成可"
```

SQLFluff (4.1.0)

[error] Do not use special characters in identifiers.

(RF05)